modules/storage/storage-accounts/.test/encr/main.test.bicep

```diff
@@ -87,12 +87,10 @@ module testDeployment '../../main.bicep' = {
       containerDeleteRetentionPolicyDays: 10
       containerDeleteRetentionPolicyAllowPermanentDelete: true
       defaultServiceVersion: '2008-10-27'
-      deleteRetentionPolicy: true
+      deleteRetentionPolicyEnabled: true
       deleteRetentionPolicyDays: 9
       isVersioningEnabled: true
       lastAccessTimeTrackingPolicyEnable: true
-      restorePolicyEnabled: true
-      restorePolicyDays: 8
```

Comment:
Removed as the default for the new parameter `allowPermanentDelete` is `true`:

https://learn.microsoft.com/en-us/azure/templates/microsoft.storage/storageaccounts/blobservices?pivots=deployment-language-bicep#deleteretentionpolicy

Reply:
testing again with changing the conditions of in the module to be set to `null` if set to false.